index.html

```diff
@@ -237,12 +283,22 @@ <h5 class="mb">Would you like to know more information on bentos around you?</h5
 
 		    $(window).scroll(function () {
 		        EasyPeasyParallax();
+		        if ($('#bento-view-info').isOnScreen() && !isCountUpAnimated) {
+		        	console.log("Hey Buddy")
+		        	// get mixpanel event info
+		        	if (isViewDataSet) {
+				 	   $('#bento-view-info').countUpAnimate(0, currentView);
+				    	isCountUpAnimated = true;		        		
```

Comment:
maybe this `isCountUpAnimated` flag would be better set inside the countUpAnimate function itself? just a suggestion